xmtp_mls/src/configuration.rs

```diff
@@ -55,6 +55,7 @@ pub const DEFAULT_GROUP_NAME: &str = "";
 pub const DEFAULT_GROUP_DESCRIPTION: &str = "";
 pub const DEFAULT_GROUP_IMAGE_URL_SQUARE: &str = "";
 pub const DEFAULT_GROUP_PINNED_FRAME_URL: &str = "";
+pub const DEFAULT_MESSAGE_EXPIRATION_MS: i64 = 0;
```

Comment:
Still thinking of setting this to null instead of 0, if you have another idea, lmk.

Reply:
@cameronvoell might have thoughts on this.

Reply:
Since its a value in the map, instead of keeping a default value I decided to not insert it at all.

Reply:
We had issues in the past of if two people are using two different versions of the SDK in the same group chat. We need to make sure that the person on the old version of the sdk does not break. I believe there was some sort of migration that needs to happen but again @cameronvoell would remember better than me how we decided to go about this for new metadata.

Reply:
No migration needed here, existing groups will not have message expiration set, but updating the expiration set will be possible with default permission of admin_only (for groups created before this change goes in) => 

https://github.com/xmtp/libxmtp/blob/ab9bb055e6fe60057836f5a631733d2aee2af584/xmtp_mls/src/groups/group_permissions.rs#L1041-L1049

That path of adding new metadata to existing groups was tested with this change here, which we can reference for adding expiration here: https://github.com/xmtp/libxmtp/pull/888/files